apps/mobile/src/screens/signup/signup.screen.tsx

```diff
@@ -0,0 +1,21 @@
+import { Heading, Stack, Text } from 'native-base';
+
+import { SignUpForm } from './compose';
+import SafeArea from '../../components/safe-area';
+import theme from '../../theme';
+
+const SignUp = () => {
```

Comment:
I found a couple of problems on this screen:

1. There is no way to go back to the sign-in screen. Please add the left arrow to the top of the screen, similar to what we have on the profile  screen.
2. On iOS, the keyboard overlaps the form. Try to use the existing `SafeArea` component.
![Simulator Screenshot - iPhone 14 Pro Max - 2023-10-12 at 12 28 10](https://github.com/animavita/animavita/assets/19699724/881bd783-ff83-4b32-8543-f63cc5039b65)

Reply:
@Isaac-alencar instead of trying to solve this, what about making a multistep form? Like we have in the adoption form. On mobile apps, it's common to split big forms into separate steps. I think it's more user-friendly.

Reply:
Thank you for reviewing! sounds good to me, I'll peek at the adoption form and implement the same approach